metricbeat/docs/modules/docker.asciidoc

```diff
@@ -7,6 +7,9 @@ This file is generated! See scripts/docs_collector.py
 
 This module fetches metrics from https://www.docker.com/[Docker] containers. The default metricsets are: `container`, `cpu`, `diskio`, `healthcheck`, `info`, `memory` and `network`. The `image` metricset is not enabled by default.
 
+[float]
+=== Compability
+
 The Docker module is currently not tested on Windows.
```

Comment:
Should we have any details around with which docker version we test?

Reply:
Added more details, though the versions are the ones in travis and jenkins.